docs/tutorials/multimodal/object_detection/quick_start/quick_start_coco.md

```diff
@@ -0,0 +1,204 @@
+# AutoMM Detection - Quick Start on a Tiny COCO Format Dataset
+:label:`sec_automm_detection_fast_finetune_coco`
+
+In this section, our goal is to fast finetune a pretrained model on a small dataset in COCO format, 
+and evaluate on its test set. Both training and test sets are in COCO format.
+See :ref:`sec_automm_detection_convert_to_coco` for how to convert other datasets to COCO format.
+
+To start, let's import MultiModalPredictor:
+
+```python
+from autogluon.multimodal import MultiModalPredictor
+```
+
+And also import some other packages that will be used in this tutorial:
+
+```python
+import os
+import time
+
+from autogluon.core.utils.loaders import load_zip
+```
+
+We have the sample dataset ready in the cloud. Let's download it:
+
+```python
+zip_file = "s3://automl-mm-bench/object_detection_dataset/tiny_motorbike_coco.zip"
+download_dir = "./tiny_motorbike_coco"
+
+load_zip.unzip(zip_file, unzip_dir=download_dir)
+data_dir = os.path.join(download_dir, "tiny_motorbike")
+train_path = os.path.join(data_dir, "Annotations", "coco_trainval.json")
```

Comment:
Can we change the name to `tiny_motorbike_coco_trainval.json`? Because if we just say `coco_trainval`, it might make people think we are using COCO dataset.